numba/tests/test_random.py

```diff
@@ -1374,6 +1380,92 @@ def test_dirichlet_exceptions(self):
                 str(raises.exception),
             )
 
+class TestRandomNoncentralChiSquare(BaseTest):
+
+    def _check_sample(self, size, sample):
+
+        """Check output structure"""
```

Comment:
Using triple quotes `"""` is commonly associated with docstrings rather than inline comments. I would recommend to us the hash-symbol instead: `#`.